### PR TITLE
issue #65,#66

### PR DESCRIPTION
**Pull request by owy1**
issue #66 placed url under "Snohomish County"
issue #65 placed url under "Yakima County"

### CODE REVIEW

File: _data/data.yml [hunk 1]
```diff
@@ -133,9 +133,23 @@
       - title: San Juan County GIS open data portal
         url: http://data.sjcgis.opendata.arcgis.com/
 
+    - name: Snohomish County
+      cities:
+        - name: Bothell
+          links:
+          - title: City of Bothell GIS Data
```
Comment (sethvincent) [inline _data/data.yml]: This line and the one below it (lines 140 & 141) should be intended with two more spaces.

File: _data/data.yml [hunk 2]
```diff
@@ -133,9 +133,23 @@
...
         - name: Bellingham
           links:
           - title: City of Bellingham
             url: http://cob.org/services/maps/
+
+    - name: Yakima County
+      cities:
+        - name: Yakima
+          links:
+          - title: City of Yakima Open Data
```
Comment (sethvincent) [inline _data/data.yml]: Like above, lines 154 and 155 should be indented with two more spaces.

**Review by sethvincent — CHANGES_REQUESTED** [review]

**Comment by sethvincent** [comment]
Thanks @owy1! Just a couple little formatting fixes to change if you can, and then I'll merge it.

**Comment by owy1** [comment]
per request, commit e32576ec04c9003b01ab9f969d6a9d224a7fe1c7

**Review by sethvincent — APPROVED** [review]

**Comment by sethvincent** [comment]
Awesome, thanks!

**Pull request merged**